web/global.css

```diff
@@ -1,4 +1,5 @@
-@import "tailwindcss" source(".");
+@import "tailwindcss";
+@source ".";
```

Comment:
why splitting it to a separate line、

Reply:
![image](https://github.com/user-attachments/assets/070111ee-d4b8-4db5-bb96-fb8b63b16956)

it shows as a syntax error in my devenv